### PR TITLE
Check Webchat Status response

### PR DESCRIPTION
**Pull request by severinbeauvais**
*Issue #:* /bcgov/entity#7031

*Description of changes:*
- check status response for whether to disable Chat button
- updated app version

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the namerequest license (Apache 2.0).

### CODE REVIEW

File: src/components/common/chat-popup.vue [hunk 1]
```diff
@@ -67,10 +67,12 @@ export default class ChatPopup extends Vue {
...
       this.chatStatus = await axios
         .get(this.webChatStatusUrl)
-        .then(response => (this.chatStatus = response.data.status))
+        .then(response => {
+          return response.data?.status || 'response error'
+        })
         .catch(error => {
           console.error('failed to get webchat status, error =', error) // eslint-disable-line no-console
-          this.chatError = true
+          return error.response?.statusText || 'network error'
```
Comment (jordiwes) [inline src/components/common/chat-popup.vue:75]: do we need to call this every so often in case someone is sitting on this page?

Comment (severinbeauvais) [reply src/components/common/chat-popup.vue:75]: Probably a good idea but not part of the requirements at this time :/

**Review by jordiwes — APPROVED** [review]
approved with comments :)

**Review by cameron-freshworks — APPROVED** [review]
LGTM

File: src/components/common/chat-popup.vue (hunk 1 above)
Comment (severinbeauvais) [reply src/components/common/chat-popup.vue:75]: https://github.com/bcgov/entity/issues/7031#issuecomment-815195154

**Pull request merged**